src/main/utility/disable_aslr.c

```diff
@@ -0,0 +1,20 @@
+#include "disable_aslr.h"
+
+#include <errno.h>
+#include <string.h>
+
+#include <sys/personality.h>
+
+#include "support/logger/logger.h"
+
+void disable_aslr() {
+    int prev_persona = personality(ADDR_NO_RANDOMIZE);
+
+    if (prev_persona != -1) {
+        debug("ASLR disabled for processes forked from this parent process.");
```

Comment:
Could we please change this to a `message` log? It's nice to get confirmation in the log file that features like this one are indeed enabled as expected.

Reply:
fixed!